site_source_files/themes/selenium/layouts/structure/section.html

```diff
@@ -211,6 +211,9 @@ <h2>Structure</h2>
                             <li>
                                 <a href="https://github.com/Harsha509">@Harsha509</a>
                             </li>
+                            <li>
```

Comment:
Please remove this for now, there is a process to become a committer, which is described here Indentation https://www.selenium.dev/governance/.

Committers are added after they fulfill this process.

Reply:
My fault. Removed it.